test/setup/iam.tf

```diff
@@ -16,8 +16,7 @@
 
 locals {
   int_required_roles = [
-    "roles/resourcemanager.projectIamAdmin",
-    "roles/storage.admin",
+    "roles/owner"
```

Comment:
nit: let's add a todo to change this to granular roles

Reply:
Done.